### ISSUE TITLE
Colons in strings

### ISSUE THREAD

**Issue by bladnman** [joeyjiron06/intellij-auto-align#10]
I have another one of these tricky colon issues
`array.unshift('MockMediaPlayer :');`
It is finding that colon in the string and aligning it
`array.unshift('MockMediaPlayer       : ');`

It's a tricky business to determine is something is wrapped (which would be best case) but at least checking the line for ' or " prior to the found element and disqualifying may help

**Comment by joeyjiron06** [comment]
Good catch, this one i've been meaning to get to. It's been on my backlog for a while. Working on it today, and maybe finish it today or tomorrow.

**Comment by joeyjiron06** [comment]
fixed in the latest release. boy this was a tricky one for sure. but got it working. let me know if it works for you :)
